Packs/F5LTM/pack_metadata.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "F5 LTM",
+    "description": "F5 Local Traffic Manager (LTM)",
+    "support": "community",
+    "currentVersion": "1.0.0",
+    "author": "Ayman Mahmoud",
+    "url": "amahmoud@paloaltonetworks.com",
```

Comment:
Hi please set this email inside the email field see [this](https://xsoar.pan.dev/docs/packs/packs-format#pack_metadatajson)

Reply:
@ShahafBenYakir , I have done that based on a previous request to replace the livecommunity url with my email

Reply:
I did update it now

Reply:
@ShahafBenYakir the validation failed because of the missing url key in the pack metadata, please let me know what to use as a url

Reply:
I did use my Github Repo in the URL key

Reply:
Yeah should've not deleted it

Reply:
@ShahafBenYakir done, I changed it to empty